test/Microsoft.AspNet.Mvc.Localization.Test/HtmlLocalizerTest.cs

```diff
@@ -150,9 +152,168 @@ public void HtmlLocalizer_HtmlWithInvalidResourcestring_ThrowsException(string f
             Assert.NotNull(exception);
             Assert.Equal("Input string was not in a correct format.", exception.Message);
         }
+
+        [Fact]
+        public void HtmlLocalizer_GetString_ReturnsLocalizedString()
+        {
+            // Arrange
+            var stringLocalizer = new TestStringLocalizer();
+
+            var htmlLocalizer = new HtmlLocalizer(stringLocalizer, new CommonTestEncoder());
```

Comment:
`CommonTestEncoder` doesn't exist. Need to get new packages, rebase, and all that stuff.